core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -1120,6 +1119,19 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     assert(redactedCmdArgMap("spark.sensitive.property") === Utils.REDACTION_REPLACEMENT_TEXT)
   }
 
+  test("redact sensitive information in sequence of key value pairs") {
```

Comment:
Sorry, but what's new in this test case? The existing test cases seems to cover these, doesn't they?
```
test("redact sensitive information")
test("redact sensitive information in command line args")
```

Reply:
If the last two cases are meaningful, shall we have the last two only?

Reply:
Agree first 2 cases are somewhat duplicating, however this test is the more direct/pure unit testing for `Utils.redact`, it spells out the exact contract for it. The other 2 tests seem to test other APIs while eventually call Utils.redact.